server/src/main/java/org/elasticsearch/index/IndexSortConfig.java

```diff
@@ -302,6 +302,15 @@ private static void validateIndexSortField(SortField sortField) {
         }
     }
 
+    public boolean hasSortOnFiled(final String fieldName) {
```

Comment:
This change will not be here once we merge https://github.com/elastic/elasticsearch/pull/121751